tests/regression/nokiasrl/nokiasrl_test.py

```diff
@@ -0,0 +1,434 @@
+            "destination-ip": {
+              "prefix": "2001:4860:8000::5/128"
+            }
+          },
+          "sequence-id": 5
+        }
+      ]
+    }
+}
+]
+"""
+
+GOOD_JSON_SPORT = """
+[
+{
+    "ipv4-filter": {
+      "entry": [
+        {
+          "action": {
+            "accept": {}
+          },
+          "match": {
+            "protocol": 6,
+            "source-port": { "value": 53 }
+          },
+          "sequence-id": 5
+        }
+      ]
+    }
+}
+]
+"""
+
+GOOD_JSON_DPORT = """
+[
+{
+    "ipv4-filter": {
+      "entry": [
+        {
+          "action": {
+            "accept": {}
+          },
+          "match": {
+            "protocol": 6,
+            "destination-port": { "value": 53 }
+          },
+          "sequence-id": 5
+        }
+      ]
+    }
+}
+]
+"""
+
+GOOD_JSON_MULTI_PROTO_DPORT = """
+[
+{
+    "ipv4-filter": {
+      "entry": [
+        {
+          "action": {
+            "accept": {}
+          },
+          "match": {
+            "protocol": 17,
+            "source-port": { "range": { "start": 1024, "end": 65535 } },
+            "destination-port": { "range": { "start": 1024, "end": 65535 } }
+          },
+          "sequence-id": 5
+        },
+        {
+          "action": {
+            "accept": {}
+          },
+          "match": {
+            "protocol": 6,
+            "source-port": { "range": { "start": 1024, "end": 65535 } },
+            "destination-port": { "range": { "start": 1024, "end": 65535 } }
+          },
+          "sequence-id": 10
+        }
+      ]
+    }
+}
+]
+"""
+
+GOOD_JSON_EVERYTHING = """
+[
+{
+    "ipv4-filter": {
+      "entry": [
+        {
+          "action": {
+            "accept": {}
+          },
+          "match": {
+            "protocol": 17,
+            "destination-ip": { "prefix": "10.2.3.4/32" },
+            "destination-port": { "value": 53 },
+            "source-ip": { "prefix": "10.2.3.4/32" }
+          },
+          "sequence-id": 5
+        },
+        {
+          "action": {
+            "accept": {}
+          },
+          "match": {
+            "protocol": 6,
+            "destination-ip": { "prefix": "10.2.3.4/32" },
+            "destination-port": { "value": 53 },
+            "source-ip": { "prefix": "10.2.3.4/32" }
+          },
+          "sequence-id": 10
+        }
+      ]
+    }
+}
+]
+"""
+GOOD_HEADER_INET6 = """
+header {
+  comment:: "The general policy comment."
+  target:: nokiasrl good-name-v6 inet6
+}
+"""
+
+# Print a info message when a term is set to expire in that many weeks.
+# This is normally passed from command line.
+EXP_INFO = 2
+
+TEST_IPS = [nacaddr.IP('10.2.3.4/32'), nacaddr.IP('2001:4860:8000::5/128')]
+
+
+_TERM_SOURCE_TAGS_LIMIT = 30
+_TERM_TARGET_TAGS_LIMIT = 70
+_TERM_PORTS_LIMIT = 256
+
+
+class NokiaSRLTest(absltest.TestCase):
+    def setUp(self):
+        super().setUp()
+        self.naming = mock.create_autospec(naming.Naming)
+
+    def _StripAclHeaders(self, acl):
```

Comment:
Is this used? I am missing seeing where it is.

Reply:
It's not, I copied it from the OpenConfig test code. Removed in both